src/SwedbankPay.Sdk.Tests/PaymentOrderTests.cs

```diff
@@ -101,7 +101,8 @@ public async Task CreatePaymentOrder_ShouldReturnPaymentOrderWithCorrectAmount()
         }
 
 
-        [Fact]
+        //TODO : Fix whenever possible
+        [Fact(Skip = "MobilePay does not support merchants having Checkout and Payment Pages")]
```

Comment:
@asbjornu this is an internal problem where authorization failes when accessing MobilePay when we use tokens having both Checkout and Payment Pages, I'm looking into a way around it.

Reply:
Yes, I think we need to support different token and `payeeId` depending on whether it's Checkout or Payments we're testing.